ReactWindows/ReactNative/CompiledReactDelegateFactory.cs

```diff
@@ -0,0 +1,299 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.Reflection;
+using System;
+using System.Globalization;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using static System.FormattableString;
+
+namespace ReactNative.Bridge
+{
+    /// <summary>
+    /// A delegate factory that will compile a delegate to call the native method.
+    /// </summary>
+    public sealed class CompiledReactDelegateFactory : ReactDelegateFactoryBase
```

Comment:
>public [](start = 4, length = 6)

Not sure why this file is showing up as "new", don't want to lose the commit history on this file...